diff --git a/general/package/Config.in b/general/package/Config.in
index 5ddbe29..bcbc040 100644
--- a/general/package/Config.in
+++ b/general/package/Config.in
@@ -96,6 +96,7 @@ source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl8733bu-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl8811cu-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl8812au-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl8812au/Config.in"
+source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl88x2cu/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl88x2eu-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rtw-hostapd/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/rubyfpv/Config.in"
@@ -108,7 +109,6 @@ source "$BR2_EXTERNAL_GENERAL_PATH/package/siproxd-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssv615x-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssv635x-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssw101b/Config.in"
-source "$BR2_EXTERNAL_GENERAL_PATH/package/txw8301-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uacme-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uclibc-compat/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uqmi-openipc/Config.in"
