diff --git a/general/package/Config.in b/general/package/Config.in
index 360aed9..be03fb7 100644
--- a/general/package/Config.in
+++ b/general/package/Config.in
@@ -67,6 +67,8 @@ source "$BR2_EXTERNAL_GENERAL_PATH/package/linux-patcher/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/majestic-fonts/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/majestic-webui/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/majestic/Config.in"
+source "$BR2_EXTERNAL_GENERAL_PATH/package/rtl88x2cu/Config.in"
+source "$BR2_EXTERNAL_GENERAL_PATH/package/waybeam_venc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/matter/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/mavfwd/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/mbedtls-openipc/Config.in"
@@ -110,7 +112,6 @@ source "$BR2_EXTERNAL_GENERAL_PATH/package/siproxd-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssv615x-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssv635x-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/ssw101b/Config.in"
-source "$BR2_EXTERNAL_GENERAL_PATH/package/txw8301-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uacme-openipc/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uclibc-compat/Config.in"
 source "$BR2_EXTERNAL_GENERAL_PATH/package/uqmi-openipc/Config.in"
