diff --git a/general/package/wifibroadcast-ng/wifibroadcast-ng.mk b/general/package/wifibroadcast-ng/wifibroadcast-ng.mk
index 382a1b5b..ac58858e 100644
--- a/general/package/wifibroadcast-ng/wifibroadcast-ng.mk
+++ b/general/package/wifibroadcast-ng/wifibroadcast-ng.mk
@@ -8,7 +8,7 @@ WIFIBROADCAST_NG_VERSION = ae2122c1fb70cd215bbc2e1b517b7b91f9401220
 WIFIBROADCAST_NG_SITE = $(call github,svpcom,wfb-ng,$(WIFIBROADCAST_NG_VERSION))
 WIFIBROADCAST_NG_LICENSE = GPL-3.0
 
-WIFIBROADCAST_NG_DEPENDENCIES += libpcap libsodium
+WIFIBROADCAST_NG_DEPENDENCIES += libpcap libsodium libevent-openipc
 
 define WIFIBROADCAST_NG_BUILD_CMDS
 	$(MAKE) CC=$(TARGET_CC) CXX=$(TARGET_CXX) LDFLAGS=-s -C $(@D) all_bin
